src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -0,0 +1,464 @@
+        Default is :math:`(a, b, c, d)=(0.5, 0.05, 0.5, 0.05)`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the logistic IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return lv_rhs(t, y, params)
+
+    def jac(t, y):
+        return lv_jac(t, y, params)
+
+    return IVP(timespan, initrv, rhs, jac)
+
+
+def lv_rhs(t, y, params):
+    """RHS for Lotka-Volterra."""
+    a, b, c, d = params
+    y1, y2 = y
+    return np.array([a * y1 - b * y1 * y2, -c * y2 + d * y1 * y2])
+
+
+def lv_jac(t, y, params):
+    """Jacobian for Lotka-Volterra."""
+    a, b, c, d = params
+    y1, y2 = y
+    return np.array([[a - b * y2, -b * y1], [d * y2, -c + d * y1]])
+
+
+def seir(timespan, initrv, params=(0.3, 0.3, 0.1, 1e7)):
+    r"""Initial value problem (IVP) based on the SEIR model.
+
+    The SEIR model with no vital dynamics is defined through
+
+    .. math::
+
+        f(t, y) =
+        \begin{pmatrix}
+            \frac{-\beta y_1 y_3}{N} \\
+            \frac{\beta y_1 y_3}{N} - \alpha y_2 \\
+            \alpha y_2 - \gamma y_3 \\
+            \gamma y_3
+        \end{pmatrix}
+
+    for some parameters :math:`(\alpha, \beta, \gamma, N)`.
+    N is a constant population count.
+    Default is :math:`(\alpha, \beta, \gamma, N)=(0.3, 0.3, 0.1, 10^7)`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(4, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`4`-dimensional mean vector and
+        :math:`4 \times 4`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float, float, float, float), optional
+        Parameters :math:`(\alpha, \beta, \gamma, N)` for the SEIR model IVP.
+        Default is :math:`(\alpha, \beta, \gamma, N)=(0.3, 0.3, 0.1, 10^7)`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the SEIR model IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return seir_rhs(t, y, params)
+
+    return IVP(timespan, initrv, rhs)
+
+
+def seir_rhs(t, y, params):
+    alpha, beta, gamma, N = params
+    y1, y2, y3, y4 = y
+    y1_next = -beta * y1 * y3 / N
+    y2_next = beta * y1 * y3 / N - alpha * y2
+    y3_next = alpha * y2 - gamma * y3
+    y4_next = gamma * y3
+
+    return np.array([y1_next, y2_next, y3_next, y4_next])
+
+
+def rigidbody(timespan, initrv):
+    r"""Initial value problem (IVP) for rigid body dynamics without external forces
+
+    The rigid body dynamics without external forces is defined through
+
+    .. math::
+
+        f(t, y) =
+        \begin{pmatrix}
+            y_2 y_3 \\
+            -y_1 y_3 \\
+            -0.51 \cdot y_1 y_2
+        \end{pmatrix}
+
+    The ODE system has no parameters.
+
```

Comment:
dont you say with the other ODEs that the problem includes a jacobian? Not too important, though

Reply:
Right, I'll fix that. Thanks!